lib/appsignal/transaction.rb

```diff
@@ -123,6 +135,11 @@ def store(key)
       @store[key]
     end
 
+    def params
+      return @params if defined?(@params)
+      request_params
```

Comment:
I don't want to set `@params` with the request params. The `@params` instance variable is only for the custom params. Otherwise you wouldn't know after this method is called if custom params are set or not.